<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IncluirLivro</class>
 <widget class="QDialog" name="IncluirLivro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Incluir Livro</string>
  </property>
  <widget class="QDialogButtonBox" name="botao_ok_livro">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>160</y>
     <width>156</width>
     <height>24</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="tit_incluirlivro">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>300</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>INCLUIR NOVO LIVRO</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lab_nomelivro">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_precolivro">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Pre√ßo:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_autorlivro">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Autor:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="recebe_nome_livro">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="recebe_preco_livro">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="recebe_autor_livro">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <zorder>tit_incluirlivro</zorder>
  <zorder>lab_nomelivro</zorder>
  <zorder>lab_precolivro</zorder>
  <zorder>lab_autorlivro</zorder>
  <zorder>recebe_nome_livro</zorder>
  <zorder>recebe_preco_livro</zorder>
  <zorder>recebe_autor_livro</zorder>
  <zorder>botao_ok_livro</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>botao_ok_livro</sender>
   <signal>accepted()</signal>
   <receiver>IncluirLivro</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botao_ok_livro</sender>
   <signal>rejected()</signal>
   <receiver>IncluirLivro</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
